src/util/balance.ts

```diff
@@ -3,8 +3,8 @@
 // of the Apache-2.0 license. See the LICENSE file for details.
 import { formatBalance } from '@polkadot/util';
 
-// Convert a `BigInt` into a nicely formatted ETH balance string.
-export function toEthBalance(v: BigInt) {
+// Convert a `BigNumber` into a nicely formatted ETH balance string.
+export function toEthBalance(v: string) {
```

Comment:
TS was complaining (in Eth2Sub) that we couldn't pass an `ethers.BigNumber` to a `BigInt` so I had to switch to string for now.